src/griffe/dataclasses.py

```diff
@@ -428,6 +432,22 @@ def is_kind(self, kind: str | Kind | set[str | Kind]) -> bool:
             kind = Kind(kind)
         return self.kind is kind
 
+    @cached_property
+    def inherited_members(self) -> dict[str, Alias]:
+        """Members that are inherited from base classes."""
+        if not isinstance(self, Class):
+            return {}
+        inherited_members = {}
+        for base in reversed(self.mro()):
+            for name, member in base.members.items():
+                inherited_members[name] = Alias(name, member, parent=self)  # type: ignore[arg-type]
```

Comment:
Inherited members are stored (cached property) as aliases. This is important for back-pointers. We cannot store the targets (actual members) into the subclasses' members, because that would overwrite their parent attribute, making the object tree incorrect.